compiler/luci-interpreter/src/kernels/DepthwiseConv2D.cpp

```diff
@@ -58,6 +59,17 @@ void DepthwiseConv2D::configure()
   {
     LUCI_INTERPRETER_CHECK(bias() == nullptr || bias()->element_type() == DataType::S32);
   }
+  else if (input()->element_type() == DataType::S8 && filter()->element_type() == DataType::S8)
+  {
+    LUCI_INTERPRETER_CHECK(filter()->shape().num_dims() == 4);
```

Comment:
This LUCI_INTERPRETER_CHECK may be redundant since there is the same check below in the code:
```
const Shape &filter_shape = filter()->shape();
LUCI_INTERPRETER_CHECK(input_shape.num_dims() == 4 && filter_shape.num_dims() == 4);
```

Reply:
I'll leave this check instead, because I think it is better to report the problem early